docs/usage/validators.md

```diff
@@ -7,12 +7,15 @@ _(This script is complete, it should run "as is")_
 
 A few things to note on validators:
 
-* validators are "class methods", the first value they receive here will be the `UserModel` not an instance
-  of `UserModel`
-* their signature can be `(cls, value)` or `(cls, value, values, config, field)`. Any subset of
-  `values`, `config` and `field` is also permitted, eg. `(cls, value, field)`, however due to the way
-  validators are inspected, the variadic key word argument ("``**kwargs``") **must** be called `kwargs`.
-* validators should either return the new value or raise a `ValueError`, `TypeError`, or `AssertionError`
+* validators are "class methods", so the first argument value they receive is the `UserModel` class, not an instance
+  of `UserModel`.
+* the second argument is always the field value to validate; it can be named as you please
+* you can also add any subset of the following arguments to the signature (the names **must** match):
+    * `values`: a dict containing the name-to-value mapping of any previously-validated fields 
+    * `config`: the model config
+    * `field`: the field being validated
+    * `**kwargs`: if provided, this will include the arguments above not explicitly listed in the signature  
+* validators should either return the parsed value or raise a `ValueError`, `TypeError`, or `AssertionError`
```

Comment:
@samuelcolvin Could you confirm that the changes I made to this section are all correct? In particular, I believe all variable names after `cls` and `value` have to be exact name matches; this was only called out for `kwargs` before (which I think was a little confusing)